src/content/pages/sprints.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: EuroPython 2024 Sprints
+subtitle: Organise or join a sprint in EuroPython 2024!
+social_card: sprints.png
+---
+
+import { Image } from "astro:assets";
+
+import sprintImage from "./images/sprint_collection_2420x1619.png";
+
+# EuroPython 2024 Sprints
+
+EuroPython 2024 Sprints will be during the weekend 13-14 July. This year, we are
+hosting the sprint weekend at a **different venue** and it will be free for
+anyone to join!
+
+As per our tradition, the conference organisers will provide the rooms and
+facilities but the sprints are organised by **YOU**. It is a great chance to
+contribute to open-source projects large and small, learn from each other, geek
+out and have fun. 🐍
+
+Lunch and coffee will be provided.
+
+**When**: 13-14 July 2024 (09:30 - 17:00)
+
+**Where**: Rajska Building (RB),
+[Žižkov Campus, VŠE](https://www.vse.cz/english/about-vse/profile/campus/)
```

Comment:
```suggestion
[Žižkov Campus, VŠE](https://maps.app.goo.gl/azRTAczu8B5ma1XFA)
```

Reply:
@cybit Can you check if this is the right place? Searching "Rajska Building (RB), Žižkov Campus, VŠE (Prague University of Economics and Business)" gives the wrong place:

![image](https://github.com/EuroPython/website/assets/75242929/42412234-1471-48db-aa78-fc9b331aaa3e)


The link I added is the right place I believe:

![image](https://github.com/EuroPython/website/assets/75242929/285dc792-9d49-44fb-9cd5-0c093f4837d5)

Reply:
@egeakman Almost. The entrance is in between the two pictures. Where the blue square is.

Reply:
@cybit Can you generate a new google maps link? I'm not sure where the exact entrance was, and don't trust my memory that much :D

Reply:
I just wanted to mention that Mia got quite a lot of details on how to get there last year. You can see it here: https://ep2023.europython.eu/where/#v%C5%A1e---sprints-venue.

Reply:
> I just wanted to mention that Mia got quite a lot of details on how to get there last year. You can see it here: https://ep2023.europython.eu/where/#v%C5%A1e---sprints-venue.

Yes. It was my silliness at the time to not the check that page first :facepalm: I only checked the sprints page and that was all

Reply:
Pointing out to ``/where/#v%C5%A1e---sprints-venue`` in the ``/sprints`` page might be a good idea though

Reply:
Absolutely! I think we just forgot about it last year!

Reply:
Tried to incorporate Mias text from last year.